src/unistd.rs

```diff
@@ -2334,3 +2334,29 @@ mod setres {
         Errno::result(res).map(drop)
     }
 }
+
+libc_bitflags!{
+    /// Options for access()
+    pub struct AccessFlags : c_int {
+        /// Test for existence of file.
+        F_OK;
+        /// Test for read permission.
+        R_OK;
+        /// Test for write permission.
+        W_OK;
+        /// Test for execute (search) permission.
+        X_OK;
+    }
+}
+
+/// Checks the file named by `path` for accessibility according to the flags given by `amode`
+/// See [access(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/access.html)
+#[inline]
```

Comment:
No need for `#[inline]`.  It's better to allow the compiler to decide.

Reply:
Ok, i did this because many other functions in unistd.rs also use inline

Reply:
Yeah, it's an anachronism that we haven't fully purged.